src/output.rs

```diff
@@ -205,6 +205,45 @@ impl Output for Status {
     }
 }
 
+/// Using [`bool`] as the return type for [`cmd!`] will return `true` if
+/// the command returned successfully, and `false` otherwise:
+///
+/// ```
+/// use cradle::*;
+///
+/// if !cmd!(%"echo foo") {
+///     panic!("echo failed!");
+/// }
```

Comment:
I wonder if we can come up with a better example. Some command that actually can return both `true` and `false` in different circumstances, maybe?

Reply:
Good call, I used `which cargo`.